<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaleTaxCalculator</class>
 <widget class="QMainWindow" name="SaleTaxCalculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="PriceTextEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PriceLabel">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="TaxRateSpinBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>200</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="TaxRateLabel">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>202</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tax Rate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TaxCalculateButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Tax</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TotalPriceTextEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TotalPriceLabel">
    <property name="geometry">
     <rect>
      <x>77</x>
      <y>364</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Price</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
